from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(354, 375)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.centralwidget)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setObjectName("verticalLayout")
        self.textEdit = QtWidgets.QTextEdit(self.centralwidget)
        self.textEdit.setMaximumSize(QtCore.QSize(16777215, 50))
        self.textEdit.setObjectName("textEdit")
        self.verticalLayout.addWidget(self.textEdit)
        self.horizontalLayout_6 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_6.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout_6.setObjectName("horizontalLayout_6")
        self.bt_clean = QtWidgets.QPushButton(self.centralwidget)
        self.bt_clean.setObjectName("bt_clean")
        self.horizontalLayout_6.addWidget(self.bt_clean)
        self.bt_delete = QtWidgets.QPushButton(self.centralwidget)
        self.bt_delete.setObjectName("bt_delete")
        self.horizontalLayout_6.addWidget(self.bt_delete)
        self.bt_percent = QtWidgets.QPushButton(self.centralwidget)
        self.bt_percent.setObjectName("bt_percent")
        self.horizontalLayout_6.addWidget(self.bt_percent)
        self.bt_division = QtWidgets.QPushButton(self.centralwidget)
        self.bt_division.setObjectName("bt_division")
        self.horizontalLayout_6.addWidget(self.bt_division)
        self.verticalLayout.addLayout(self.horizontalLayout_6)
        self.horizontalLayout_5 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_5.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout_5.setObjectName("horizontalLayout_5")
        self.bt_seven = QtWidgets.QPushButton(self.centralwidget)
        self.bt_seven.setObjectName("bt_seven")
        self.horizontalLayout_5.addWidget(self.bt_seven)
        self.bt_eight = QtWidgets.QPushButton(self.centralwidget)
        self.bt_eight.setObjectName("bt_eight")
        self.horizontalLayout_5.addWidget(self.bt_eight)
        self.bt_nine = QtWidgets.QPushButton(self.centralwidget)
        self.bt_nine.setObjectName("bt_nine")
        self.horizontalLayout_5.addWidget(self.bt_nine)
        self.bt_multiply = QtWidgets.QPushButton(self.centralwidget)
        self.bt_multiply.setObjectName("bt_multiply")
        self.horizontalLayout_5.addWidget(self.bt_multiply)
        self.verticalLayout.addLayout(self.horizontalLayout_5)
        self.horizontalLayout_4 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_4.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout_4.setObjectName("horizontalLayout_4")
        self.bt_four = QtWidgets.QPushButton(self.centralwidget)
        self.bt_four.setObjectName("bt_four")
        self.horizontalLayout_4.addWidget(self.bt_four)
        self.bt_five = QtWidgets.QPushButton(self.centralwidget)
        self.bt_five.setObjectName("bt_five")
        self.horizontalLayout_4.addWidget(self.bt_five)
        self.bt_six = QtWidgets.QPushButton(self.centralwidget)
        self.bt_six.setObjectName("bt_six")
        self.horizontalLayout_4.addWidget(self.bt_six)
        self.bt_minus = QtWidgets.QPushButton(self.centralwidget)
        self.bt_minus.setObjectName("bt_minus")
        self.horizontalLayout_4.addWidget(self.bt_minus)
        self.verticalLayout.addLayout(self.horizontalLayout_4)
        self.horizontalLayout_3 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_3.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout_3.setObjectName("horizontalLayout_3")
        self.bt_one = QtWidgets.QPushButton(self.centralwidget)
        self.bt_one.setObjectName("bt_one")
        self.horizontalLayout_3.addWidget(self.bt_one)
        self.bt_two = QtWidgets.QPushButton(self.centralwidget)
        self.bt_two.setObjectName("bt_two")
        self.horizontalLayout_3.addWidget(self.bt_two)
        self.bt_three = QtWidgets.QPushButton(self.centralwidget)
        self.bt_three.setObjectName("bt_three")
        self.horizontalLayout_3.addWidget(self.bt_three)
        self.bt_plus = QtWidgets.QPushButton(self.centralwidget)
        self.bt_plus.setObjectName("bt_plus")
        self.horizontalLayout_3.addWidget(self.bt_plus)
        self.verticalLayout.addLayout(self.horizontalLayout_3)
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_2.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.bt_0 = QtWidgets.QPushButton(self.centralwidget)
        self.bt_0.setObjectName("bt_0")
        self.horizontalLayout_2.addWidget(self.bt_0)
        self.bt_dot = QtWidgets.QPushButton(self.centralwidget)
        self.bt_dot.setObjectName("bt_dot")
        self.horizontalLayout_2.addWidget(self.bt_dot)
        self.bt_not = QtWidgets.QPushButton(self.centralwidget)
        self.bt_not.setObjectName("bt_not")
        self.horizontalLayout_2.addWidget(self.bt_not)
        self.bt_equals = QtWidgets.QPushButton(self.centralwidget)
        self.bt_equals.setObjectName("bt_equals")
        self.horizontalLayout_2.addWidget(self.bt_equals)
        self.verticalLayout.addLayout(self.horizontalLayout_2)
        self.horizontalLayout.addLayout(self.verticalLayout)
        MainWindow.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.menuBar = QtWidgets.QMenuBar(MainWindow)
        self.menuBar.setGeometry(QtCore.QRect(0, 0, 354, 25))
        self.menuBar.setObjectName("menuBar")
        self.menuFile = QtWidgets.QMenu(self.menuBar)
        self.menuFile.setObjectName("menuFile")
        MainWindow.setMenuBar(self.menuBar)
        self.bt_stories = QtWidgets.QAction(MainWindow)
        self.bt_stories.setObjectName("bt_stories")
        self.bt_about = QtWidgets.QAction(MainWindow)
        self.bt_about.setObjectName("bt_about")
        self.menuFile.addAction(self.bt_stories)
        self.menuFile.addAction(self.bt_about)
        self.menuBar.addAction(self.menuFile.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.bt_clean.setText(_translate("MainWindow", "C"))
        self.bt_delete.setText(_translate("MainWindow", "Delete"))
        self.bt_percent.setText(_translate("MainWindow", "%"))
        self.bt_division.setText(_translate("MainWindow", "/"))
        self.bt_seven.setText(_translate("MainWindow", "7"))
        self.bt_eight.setText(_translate("MainWindow", "8"))
        self.bt_nine.setText(_translate("MainWindow", "9"))
        self.bt_multiply.setText(_translate("MainWindow", "*"))
        self.bt_four.setText(_translate("MainWindow", "4"))
        self.bt_five.setText(_translate("MainWindow", "5"))
        self.bt_six.setText(_translate("MainWindow", "6"))
        self.bt_minus.setText(_translate("MainWindow", "-"))
        self.bt_one.setText(_translate("MainWindow", "1"))
        self.bt_two.setText(_translate("MainWindow", "2"))
        self.bt_three.setText(_translate("MainWindow", "3"))
        self.bt_plus.setText(_translate("MainWindow", "+"))
        self.bt_0.setText(_translate("MainWindow", "0"))
        self.bt_dot.setText(_translate("MainWindow", "."))
        self.bt_not.setText(_translate("MainWindow", "???"))
        self.bt_equals.setText(_translate("MainWindow", "="))
        self.menuFile.setTitle(_translate("MainWindow", "File"))
        self.bt_stories.setText(_translate("MainWindow", "History"))
        self.bt_about.setText(_translate("MainWindow", "About"))
